script/Arcueid/Day 13/13_00_ARC13_5.txt

```diff
@@ -386,18 +427,18 @@ The areas that represent the source of life only exist in living things."
 [sha:ad72f821212c699f3422e6e29bced4d6884cc561]{
 -- Page 429, Offset 32896.
 -- ナイフを握る。
-I clench my knife.
+I grip my knife.
 }
 [sha:e22394029a2733626e565f7bec06f3910a5545c7]{
 -- Page 429, Offset 32897.
 -- <指先|まったん>から<心臓|ちゅうしん>まで生命として<凍結|ていし>する。
 // uh huh...... cool line...
-My life freezes from my fingertips to my heart.
+All vital activity, from my extremities all the way to my core, comes to a standstill.
```

Comment:
make it "vital activities" and "come". "comes" sounds really weird seeing as the preceding sub-clause is a list but I see that you're matching "activity".